person.rb

```diff
@@ -0,0 +1,36 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :age, :name
```

Comment:
- [ ] [Optional] It would be better professionally if you move the attributes at the top of the class even before the initialize method.

Reply:
Done! thank you for the suggestion